### ISSUE TITLE
Odd behaviors with 6.0.400 previews

### ISSUE THREAD

**Issue by baronfel** [ionide/ionide-vscode-fsharp#1748]
Users reported issues with finding reference counts (and likely many other features) on 6.0.400 previews. We should investigate before release.

> This bug is fixed in Ionide 7.0.0 with .NET Core 6.0.302 but shows up again in .NET Core 6.0.400-preview.22330.6. (Easy workaround: `local.json` to force the non-preview build).

_Originally posted by @backerman in https://github.com/ionide/ionide-vscode-fsharp/issues/1718#issuecomment-1193180067_

**Comment by pkese** [comment]
After upgrading dotnet to 6.0.400 (this is now not a preview anymore), I'm getting same behavior as described in #1732.

**Comment by pkese** [comment]
`sudo apt-get install dotnet-sdk-6.0=6.0.302-1`  
appears to solve the problem.

**Comment by babysnakes** [comment]
Indeed, working on the newly updated `6.0.400` the behavior described in #1718 is back 🙁. As a workaround I added a `global.json` file pointing to `6.0.303` and now the functionality is back.
